test-it/results/ComplexMathFunctions.vtr

```diff
@@ -1,9 +1,12 @@
 ReOrImToComplex 2.000000 0.500000 => (2 0.5)
```

Comment:
When there is an issue with tests I found the output to be more difficult to interpret than it needs to be. Please add a unique identifier at the beginning of each line, like 000, 001 and so forth, that way it is also easier to talk about a problem with somebody else.

Thanks to Sankara for previously suggesting this improvement in our tests output.